doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_cartodb_sink.md

```diff
@@ -446,8 +448,8 @@ cygnus-ngsi.sinks.cartodb-sink.enable_name_mappings = false
 cygnus-ngsi.sinks.cartodb-sink.enable_lowercase = false
 cygnus-ngsi.sinks.cartodb-sink.keys_conf_file = /usr/cygnus/conf/cartodb_keys.conf
 cygnus-ngsi.sinks.cartodb-sink.swap_coordinates = true
-cygnus-ngsi.sinks.cartodb-sink.enable_raw = true
-cygnus-ngsi.sinks.cartodb-sink.enable_distance = false
+cygnus-ngsi.sinks.cartodb-sink.enable_raw_historic = true
```

Comment:
A change in the configuration use to break backward compatibility... are the old ones still supported? Has been them mentioned in the deprecated documentation (if Cygnus has such kind of documentation?